modules/distribution/src/repository/resources/conf/deployment.toml

```diff
@@ -0,0 +1,26 @@
+[server]
```

Comment:
Missing License header.

Reply:
Since this is a configuration file all the products should have the same settings. If we need to add the license header will have to update all the products and kernel[1]

[1] - https://github.com/wso2/carbon-kernel/blob/4.5.x/distribution/kernel/carbon-home/repository/conf/deployment.toml